src/platform_interface.rs

```diff
@@ -11,7 +11,13 @@ pub(crate) trait PlatformInterface {
   ) -> Result<Command, OutputError>;
   
   /// Get generate file ext
-  fn get_recipe_file_ext(command: &str) -> String;
+  fn get_script_file_ext(command: &str) -> &str {
+      if command.ends_with("powershell") {
```

Comment:
Should this also check for `powershell.exe`?

```suggestion
      if command.ends_with("powershell") || command.ends_with("powershell.exe") {
```